### DISCUSSION TITLE
Set Updates packages to "Up-to-date" instead of unavailable

### DISCUSSION THREAD

**Discussion by walberjunior** [dougiteixeira/proxmoxve#329] [Feature request] [answered]
Today I noticed that after updating PVE the sensor Updates packages became unavailable:

![image](https://github.com/dougiteixeira/proxmoxve/assets/20650850/f75597d4-3e60-4108-92ad-f49a81a1f815)

This caused problems with some sensors that I use to check if any entity or device is offline.
Despite controlling this sensors and being able to place an exception for "binary_sensor.node_pve_updates_packages" it would be nice for the sensor to always have a value, perhaps "Up-to-date".
Maybe this applies to other sensors.

**Comment by dougiteixeira** [comment, marked as answer]
Duplicata: #311
